.github/workflows/ci-for-default-ns.yaml

```diff
@@ -75,16 +75,23 @@ jobs:
           # for testing start-debug and stop-debug deployment
           # mons 
           kubectl rook_ceph debug start rook-ceph-mon-a
-          sleep 5
+          mons=$(kubectl get pod -l app=rook-ceph-mon -n rook-ceph -o jsonpath='{.items[*].metadata.name}'| grep debug)
+          for val in $mons; do
+            if [[ $val == *"debug"* ]];
+            then
+              debug_mon=$val
+            fi  
+          done
+          timeout 60 sh -c 'until [ "$(kubectl get pod $debug_mon -n rook-ceph -o jsonpath='{.status.phase}'| grep -c "Running")" -eq 1 ]; do echo "waiting for pod rook-ceph-mon-a-debug to get created " && sleep 1; done'
           kubectl rook_ceph debug stop rook-ceph-mon-a
           # osd  
           kubectl rook_ceph debug start rook-ceph-osd-0
-          sleep 5
+          tests/github-action-helper.sh wait_for_operator_pod_to_be_ready_state_default
```

Comment:
This helper waits for the operator pod, but don't we need to wait for the osd debug pod?

Reply:
Yes added